Source/ACE.Server/WorldObjects/Player_Use.cs

```diff
@@ -59,36 +59,8 @@ public void HandleActionUseWithTarget(uint sourceObjectGuid, uint targetObjectGu
                 return;
             }
 
-
-            // Is the target the player?
-            if (targetObjectGuid == Guid.Full)
-            {
-                // using something on ourselves
-                if (sourceItem.WeenieType == WeenieType.ManaStone)
-                    ((ManaStone)sourceItem).HandleActionUseOnTarget(this, this);
-                else
-                    RecipeManager.UseObjectOnTarget(this, sourceItem, this);
-
-                return;
-            }
-
-
             // Is the target item in our possession?
-            var targetItem = FindObject(targetObjectGuid, SearchLocations.MyInventory | SearchLocations.MyEquippedItems);
-
-            if (targetItem != null)
-            {
-                if (sourceItem.WeenieType == WeenieType.ManaStone)
-                    ((ManaStone)sourceItem).HandleActionUseOnTarget(this, targetItem);
-                else
-                    RecipeManager.UseObjectOnTarget(this, sourceItem, targetItem);
-
-                return;
-            }
-
-
-            // Is the target on the landblock?
-            targetItem = FindObject(targetObjectGuid, SearchLocations.Landblock);
+            var targetItem = FindObject(targetObjectGuid, SearchLocations.MyInventory | SearchLocations.MyEquippedItems | SearchLocations.Landblock);
```

Comment:
If you add SearchLocations.Landblock here, the comment will need to be changed as the search will now include things not in the players possession